include/boost/geometry/strategies/cartesian/cart_intersect.hpp

```diff
@@ -95,6 +97,39 @@ struct relate_cartesian_segments
         return strategy_type();
     }
 
+    template <typename Geometry>
+    struct area_strategy
+    {
+        typedef area::surveyor
+            <
+                typename point_type<Geometry>::type,
+                CalculationType
+            > type;
+    };
+
+    template <typename Geometry>
+    static inline typename area_strategy<Geometry>::type get_area_strategy()
+    {
+        typedef typename area_strategy<Geometry>::type strategy_type;
+        return strategy_type();
+    }
+
+    template <typename Geometry>
+    struct distance_strategy
+    {
+        typedef distance::pythagoras
+            <
+                CalculationType
+            > type;
+    };
+
+    template <typename Geometry>
+    static inline typename distance_strategy<Geometry>::type get_distance_strategy()
+    {
+        typedef typename distance_strategy<Geometry>::type strategy_type;
+        return strategy_type();
+    }
```

Comment:
It's not used in the strategy however an algorithm which use the intersection strategy also needs area strategy. The algorithm is equals() which does this TrivialCheck comparing areas of polygons first before calculating turns.